modules/regex/tests/test_regex.h

```diff
@@ -150,6 +191,37 @@ TEST_CASE("[RegEx] Invalid end position") {
 
 	CHECK(re.sub(s, "", true, 0, 10) == "Gdt");
 }
+
+TEST_CASE("[RegEx] Get match string list") {
+	const String s = "Godot Engine";
+
+	RegEx re("(Go)(dot)");
+	Ref<RegExMatch> match = re.search(s);
+	REQUIRE(match != nullptr);
+	PackedStringArray result;
+	result.append("Godot");
+	result.append("Go");
+	result.append("dot");
+	CHECK(match->get_strings() == result);
+}
+
+TEST_CASE("[RegEx] Match start and end positions") {
+	const String s = "Whole pattern";
+
+	RegEx re1("pattern");
+	REQUIRE(re1.is_valid());
+	Ref<RegExMatch> match = re1.search(s);
+	REQUIRE(match != nullptr);
+	CHECK(match->get_start(0) == 6);
+	CHECK(match->get_end(0) == 13);
+
+	RegEx re2("(?<vowel>[aeiou])");
+	REQUIRE(re2.is_valid());
+	match = re2.search(s);
+	REQUIRE(match != nullptr);
+	CHECK(match->get_start("vowel") == 2);
+	CHECK(match->get_end("vowel") == 3);
+}
 } // namespace TestRegEx
 
-#endif // TEST_REGEX_H
+#endif // TEST_REGEX_H
```

Comment:
```suggestion
#endif // TEST_REGEX_H

```
Missing line termination

Reply:
I included the missing line termination, but github doesnt show it for some reason. Did it work?